prisma/seed/forms/checkinform.ts

```diff
@@ -158,6 +167,10 @@ export const populateCheckinForm = async () => {
                             },
                         },
                         text: "{{Yes,No}}Did you deploy to Production at the end of this Sprint?",
+                        parseConfig: {
+                            Yes: "Yes",
```

Comment:
should we have `yes/no` (instead of "Yes/No") for the keys to be consistent with the naming?